sympy/physics/control/lti.py

```diff
@@ -802,80 +918,72 @@ def doit(self, **kwargs):
         TransferFunction((2 - s**3)*(-a*p**2 - b*s), (-p + s)*(s**4 + 5*s + 6), s)
 
         """
-        res = None
-        for arg in self.args:
-            arg = arg.doit()
-            if res is None:
-                res = arg
-            else:
-                num_ = arg.num * res.num
-                den_ = arg.den * res.den
-                res = TransferFunction(num_, den_, self.var)
-        return res
+        def _SISO_doit():
```

Comment:
Whichever method you modify in Series and Parallel, update the docstring and add one example related to MIMO TF.

Reply:
I had modified the tests for this particular change.

Reply:
> I had modified the tests for this particular change.

I don't get it.  `doit()` method in `Series` and `Parallel` should include at least one example for MIMO system.

Reply:
Done.